### PR TITLE
added my name

### PR DESCRIPTION
**Pull request by amha19**
I added my name to app.js file.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1 +1,2 @@
 const textOne = 'Hello everyone';
+const name = 'Amha';
```
Comment (amted20) [inline app.js]: Please add your last name.

**Comment by amha19** [comment]
check it now

File: app.js (hunk 1 above)
Comment (amha19) [reply app.js]: I have added my last name now.

Comment (amted20) [reply app.js]: Nice

**Pull request merged**